.buildkite/pipeline.yml

```diff
@@ -0,0 +1,82 @@
+steps:
+  - label: "Nvidia GPUs -- CUDA.jl"
+    plugins:
+      - JuliaCI/julia#v1:
+          version: "1.10"
+    agents:
+      queue: "juliagpu"
+      cuda: "*"
+    env:
+      EXAMODELS_TEST_CUDA: "TRUE"
+    command: |
+      julia --color=yes -e '
+        using Pkg
+        Pkg.add("CUDA")
+        Pkg.add("KernelAbstractions")
+        Pkg.add("NLPModels")
```

Comment:
why was this necessary?

Reply:
It depends on how you want to run the tests.
If you just want to use `runtests.jl`, we can modify it.
But with the current `pipeline.yml`, we can also run specific and longer tests that target specific GPUs without environment variables.

I did something similar in `Krylov.jl`, using buildkite to run GPU tests separately from CPU tests, which is recommended to avoid occupying the GPUs for too long, allowing other users and repositories to access them.